password_validators/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```

Comment:
kira2 file ini bisa diapus gak ya(?) soalnya considered as not covered di sonar wkwk

Reply:
Bisa, will do!

Reply:
Pushed. Donezo.